articles/manual/index.md

```diff
@@ -0,0 +1,106 @@
+that runs on the adjunct processor.
+When the subroutine completes, the host program gets access to the
+subroutine's results.
+
+Q# provides a small set of primitive types, along with two ways
+(arrays and tuples) for creating new, structured types.
+It supports a basic procedural model for writing programs,
+with loops and if/then statements.
+The top-level constructs in Q# are user defined types, operations,
+and functions.
+
+Q# is scalable: it can be used to write simple demonstration programs like teleport that run on a few qubits, but also supports writing large, sophisticated programs such as simulations of complex molecules that will require large machines with millions of qubits. Even though large physical machines are still in the future, Q# allows a programmer to program complex quantum algorithms now. What is more, Q# supports various tasks such as debugging, profiling, resource estimation, and certain special-purpose simulations in a scalable way.
+
+
+# Why Q#?
+
+[![](http://img.youtube.com/vi/cOUrzxyng04/0.jpg)](http://www.youtube.com/watch?v=cOUrzxyng04 "")
+
+ While quantum hardware is still in an early developing stage, we need 
+ to build adequate software for quantum computers to take advance 
+ of the quantum power of computation when time comes. Q# is a tool 
+ made to provide a platform where developers can write and test programs 
+ to be run in a future generation of quantum computers.
+
+For more information about the foundations and 
+motivation of Q# check out 
+[Why do we need Q#?](https://devblogs.microsoft.com/qsharp/why-do-we-need-q/).
+
+# How to learn Q#?
+
+## Prerequired knowledge to use Q#
+There is no need to be an expert on quantum physics to start writing your 
+first quantum programs. However, there are some concepts that are essential 
+to understand quantum computing:
+* **Complex numbers:** quantum mechanics makes use of complex numbers to 
+express quantum states. Some experience with complex numbers is crucial to
+understand the basics of quantum computation.
+* **Linear algebra (vectors and matrices):** quantum states are vectors 
+  in a vector space over the complex numbers. Quantum computing makes 
+  intense use of linear operators acting on these states. Therefore 
+  some level of expertise in linear algebra is indispensable to use Q#.
+* **Dirac notation:** most quantum computing texts use Dirac notation 
+  to express the mathematical concepts. Dirac notation is just a simple 
+  notation for linear algebra widely used in quantum physics.
+* **Basic concepts of quantum mechanics:** wether it is not required to 
+  have a deep understanding of quantum mechanics to use Q#, it is highly 
+  recommendable to be familiar with some basic concepts, namely 
+  *quantum state*, *qubit*, *observable*, *measurement*, *Hamiltonian*,
+  *superposition*, *entanglement*, *interference*, among others.  
+## Learning materials
+Quantum computing is an emerging field and fortunately, there is an 
```

Comment:
```suggestion
Quantum computing is an emerging field and fortunately, there are an 
```

Reply:
In this one I think "is" is correct because "amount" is singular.